frontend/src/views/SecretOverviewPage/SecretOverviewPage.tsx

```diff
@@ -191,20 +191,10 @@ export const SecretOverviewPage = () => {
 
   const userAvailableEnvs = currentWorkspace?.environments || [];
 
-  const readableEnvs = userAvailableEnvs?.filter(({ slug }) =>
-    permission.can(
-      ProjectPermissionActions.Read,
-      subject(ProjectPermissionSub.Secrets, {
-        environment: slug,
-        secretPath
```

Comment:
Wouldn't this be solved if we had the same custom glob pattern defined in the frontend too? It's not that this is in correct but more so that the CASAL is not 1:1 like the backend. Or did i miss something

Reply:
Both frontend and backend has all the same operator and there is glob support. But think of this way. 

The user permission is `/backend`. But if you check permission in `/` both backend and frontend will throw an error. That's folders need to shown irrespective of both